Automattic/sensei
# POC pr: Reload table data on data update

### Note
- The PR is not in the final stage I just made it work so that I can get approval or decline on this approach
- I didn't find any use case which this can't cover and it doesn't add up to complexity
- can be easily used for sorting, pagination, everythig.
- 
### Description

After applying an action in the modal we need to update the screen.
We chose to refresh the page as the simplest solution.
However, it would be nice to update the table list without reloading the whole page.
The resulting solution might require the design as well (to express the loading state, etc.).

### Changes proposed in this Pull Request
* Override WP_List_Table **ajax_response**() function
* Add action add_action( 'wp_ajax_fetch_custom_list', array( $this, 'fetch_custom_list' ) )
* Make a call to ajaxurl when changes occur so that table data is rendered WITHOUT refreshing

### Testing instructions
- Open student's table
- Change data using user action when the modal is closed data should be updated without page refresh
